build.py

```diff
@@ -2683,6 +2710,19 @@ def enable_all():
         default=DEFAULT_TRITON_VERSION_MAP["rhel_py_version"],
         help="This flag sets the Python version for RHEL platform of Triton Inference Server to be built. Default: the latest supported version.",
     )
+    parser.add_argument(
+        "--build-secret",
```

Comment:
Does the name of the flag correspond the action? Seems it designed to add `--build-argument` to the `docker build` ?